quickjs.c

```diff
@@ -47440,6 +47482,35 @@ static JSValue js_array_buffer_get_byteLength(JSContext *ctx,
     return JS_NewUint32(ctx, abuf->byte_length);
 }
 
+// #sec-get-arraybuffer.prototype.resizable
+static JSValue js_array_buffer_get_resizable(JSContext *ctx,
+                                             JSValueConst this_val,
+                                             int class_id)
+{
+    JSArrayBuffer *abuf = JS_GetOpaque2(ctx, this_val, class_id);
+    if (!abuf)
+        return JS_EXCEPTION;
+    if (abuf->shared) {
+        return JS_ThrowTypeError(ctx, "resizable called on SharedArrayBuffer");
+    }
+    return JS_NewBool(ctx, abuf->resizable);
+}
+
+// #sec-get-arraybuffer.prototype.maxbytelength
+static JSValue js_array_buffer_get_maxByteLength(JSContext *ctx,
+                                                 JSValueConst this_val,
+                                                 int class_id)
+{
+    JSArrayBuffer *abuf = JS_GetOpaque2(ctx, this_val, class_id);
+    if (!abuf)
+        return JS_EXCEPTION;
+    if (abuf->shared) {
+        return JS_ThrowTypeError(ctx, "resizable called on SharedArrayBuffer");
+    }
+    /* return 0 if detached */
+    return JS_NewUint32(ctx, abuf->resizable ? abuf->max_byte_length : abuf->byte_length);
```

Comment:
Maybe this because of that resizable implication I mentioned earlier?
```suggestion
    return JS_NewUint32(ctx, abuf->max_byte_length);
```